src/Kernel32.Tests/storebanned/Kernel32Facts.cs

```diff
@@ -1130,6 +1130,43 @@ unsafe uint GetProcessIdThreadProc(void* data)
         GC.KeepAlive(threadStartRoutine); // Make sure that the delegate stays alive until the test is done
     }
 
+    [Fact]
+    public unsafe void DeviceIOControl_Works()
+    {
+        const uint IOCTL_DISK_GET_DRIVE_GEOMETRY = 0x070000;
+        const string drive = @"\\.\PhysicalDrive0";
+
+        using (var device = CreateFile(
+            filename: drive,
+            access: 0,
+            share: Kernel32.FileShare.FILE_SHARE_READ | Kernel32.FileShare.FILE_SHARE_WRITE,
+            securityAttributes: (SECURITY_ATTRIBUTES*)null,
+            creationDisposition: CreationDisposition.OPEN_EXISTING,
+            flagsAndAttributes: 0,
+            SafeObjectHandle.Null))
+        {
+            Assert.False(device.IsInvalid);
+
+            DISK_GEOMETRY pdg = default;
+
+            Assert.True(DeviceIoControl(
+                device,
+                (int)IOCTL_DISK_GET_DRIVE_GEOMETRY,
```

Comment:
A set of well-know IOCTL codes are defined here: https://docs.microsoft.com/en-us/windows/win32/api/winioctl/ .

Reply:
That's unfortunate. I filed an issue to track eventually changing the signature to `uint` when we can take breaking changes.